wrapper/src/main/java/software/amazon/jdbc/dialect/RdsMultiAzDbClusterPgDialect.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.dialect;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import software.amazon.jdbc.exceptions.ExceptionHandler;
+import software.amazon.jdbc.exceptions.MultiAzDbClusterPgExceptionHandler;
+import software.amazon.jdbc.hostlistprovider.RdsMultiAzDbClusterListProvider;
+import software.amazon.jdbc.util.DriverInfo;
+
+public class RdsMultiAzDbClusterPgDialect extends PgDialect {
+
+  private static MultiAzDbClusterPgExceptionHandler exceptionHandler;
+
+  private static final String TOPOLOGY_QUERY =
+      "SELECT LOWER(id) AS id, LOWER(endpoint) AS endpoint, port FROM "
```

Comment:
do we really need LOWER here ? I think all of the data is already in lower case

Reply:
No, we don't, now. and removed.